var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -627,6 +629,11 @@ def post_configure_fixes(self):
                         'build --action_env LD_LIBRARY_PATH="' + slibs + '"',
                         '.tf_configure.bazelrc')
 
+        filter_file('build:opt --copt=-march=native', '',
+                    '.tf_configure.bazelrc')
+        filter_file('build:opt --host_copt=-march=native', '',
+                    '.tf_configure.bazelrc')
```

Comment:
Does this have any possible negative side-effects for other compilers, or do Spack's compiler wrappers inject the right optimization flags anyway?

Reply:
Yes, following file inject the right optimization flags.
Therefore, I removed -march option .

```
lib/spack/llnl/util/cpu/microarchitecture.py
```

Reply:
Related to #12948.